### DISCUSSION TITLE
Gradle Plugin for OpenAPI Diff

### DISCUSSION THREAD

**Discussion by nbiastaunton** [OpenAPITools/openapi-diff#601] [Show and tell]
For anyone that may find it useful, a Gradle plugin has been created to allow the use of OPenAPI Diff (currently version 2.1.0-beta.8) as part of a gradle build. 

The plugin can be found at 
Code:                   https://github.com/x3tirl/openapi_diff_gradle_plugin
Gradle Plugins:   https://plugins.gradle.org/plugin/com.x3t.gradle.plugins.openapi.openapi_diff 

Examples of its use can be found in the demo sub-directory or in the README.adoc

**Comment by joschi** [comment]
Awesome, thanks for sharing! ❤️

**Reply by Praytic** [reply]
Can it be adopted as a part of this repo?
